app/components/work_packages/activities_tab/journals/new_component.sass

```diff
@@ -0,0 +1,28 @@
+#work-packages-activities-tab-journals-new-component
+    #input-trigger-column
+        width: 100%
+        button
+            background: var(--bgColor-default)
+            cursor: text
+            .Button-content
+                display: block
+    #ck-editor-column
+        width: 100%
+        @media screen and (max-width: $breakpoint-md)
+            width: calc(100% - 40px)
+    // prototypical primer style adoptions
+    .ck-toolbar
+        border-radius: var(--borderRadius-medium) var(--borderRadius-medium) 0px 0px
+    .document-editor__editable-container
+        background: var(--bgColor-default)
+        border-radius: 0px 0px var(--borderRadius-medium) var(--borderRadius-medium)
+    .ck-content
+        background: var(--bgColor-default)
+        max-height: 30vh
+        overflow-y: auto
+        border-radius: var(--borderRadius-medium)
+        margin: 5px
+    // overwrites margin bottom defined in _ckeditor.sass:13
+    opce-ckeditor-augmented-textarea .op-ckeditor--wrapper
+        margin-bottom: 0px
```

Comment:
Same as earlier, needs to be a BEM module and overrides should be generalized

Reply:
Done